### PR TITLE
UCS/ASYNC: Reduce log level of leaked event handlers

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/ucs/async/thread.c [hunk 1]
```diff
@@ -404,8 +404,8 @@ static ucs_status_t ucs_async_thread_remove_timer(ucs_async_context_t *async,
 static void ucs_async_signal_global_cleanup()
 {
     if (ucs_async_thread_global_context.thread != NULL) {
-        ucs_warn("async thread still running (use count %d)",
-                 ucs_async_thread_global_context.use_count);
+        ucs_debug("async thread still running (use count %d)",
```
Comment (dmitrygx) [inline src/ucs/async/thread.c]: q: should it be `%u` instead of `%d`?

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11635/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 07-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/b21c808f489c1e84450639cec50a6e29b800e2ec/_main) | :question: ABORTED
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/ea1b6322875920aef7af23af7ae1e80c9dd6b818/hpc-test-althca_W0) | :question: ABORTED
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/e1d8b55f6a9945b8c36d9ef3dceda330e17e2419/hpc-test-althca_W1) | :question: ABORTED
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/0dbc8980daa181085f05fe9d653cc752d186677c/hpc-test-althca_W2) | :question: ABORTED
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/3169729713034c3c325e5b1fb671ad36c8cf58f9/r-vmb-ppc-jenkins_W0) | :question: ABORTED
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/9ac9df2a807c44676efeeca8e6341b8cb69d81da/r-vmb-ppc-jenkins_W3) | :question: ABORTED
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/2b95d86cc9fcd8a601d5fbe30e9aedfb0fdb8aaf/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/2dc279d2966c4b4792363fcc29b20cd59921d1dd/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/51df51bb886add2a4be5e69da007265521e336d4/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/29d7bc35235f261768c4a2f50844f3381c1cb85e/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/ef6fb67623ee597ad95aed08fc44727427ff0081/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/a51cef26c62c4d5b1cf335a5e3386a75e53fbd91/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/39460b725007a3daa225c9e5c9e006edf978b008/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/cb9f66cac8c86b9554255b82dbec82ee88b8cc34/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/6b82445c36cb0b8b75d38c494e7e246cc6d0df9d/hpc-arm-hwi-jenkins_W3) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/1ef76c9a65b7e4b54ee5b52cc6f6fc2f62592b0b/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/3f6ddcb404ff8993a4685fae4ff16edddccded94/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/632dfdaea2b1d679036ee27f0042e2d7decd7ad4/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/08d7704fbdb4fd2ebfbae8783343c4dfcdfc2673/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/124d98a8c0fde3e2714098683d1446a142697fa8/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/bd696c8abdf86ba105c49fd1cb9c1cfc5bcfc3c7/hpc-test-node-legacy_W1) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/96accbdb16f44b8b259d1ea734c3f0809c4238e4/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/b63cbdb0ff024fb94621a22983596e80e3c60e83/hpc-test-node-legacy_W3) | :question: UNKNOWN
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/75d0182ef9072cbcd1693c744dbe320373e2deb5/hpc-test-node-new_W0) | :question: UNKNOWN
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/95fae8c8300a35b61d23af0a23c650b500ad5af6/hpc-test-node-new_W1) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/36d5db6669a2c77825018a0ebc0ac0e40354effe/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/d4d356cbde80e058067d120d472472ea59b9ff89/hpc-test-node-new_W3) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/2f5c20cc323639db3c78e7ec69eebb378a97bffb/r-vmb-ppc-jenkins_W1) | :question: UNKNOWN
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/0d45e5c777e2f07275566f2274bd42b2/raw/4eb34d091f0a5d8e12d8918aaf7a7efbc87870f4/r-vmb-ppc-jenkins_W2) | :question: UNKNOWN

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11641/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 07-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/96f2bec0d3a14b32dab6e58c3b9e820dc0b96b36/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/60823a8174c5cdd7251446db9f71e09d21a7787a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/8ac03e5d86bcfbe2c43e96f48c9a3fd3b9ddcc86/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/9eeadc754d5524bb8923e65cecbe6a50fd83d229/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/538c7f131a06435b9a584e9179e14813d4476317/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/aa64846bbe31983151e3d9c01b87ad98a11cf9f6/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/e87a3c0510a26121acd9ba5dd0d76b835f8ea11d/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/c8a47e59cae24ce13fb16586b709bcafe4cbe49f/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/34d38ea0c91264bf8d59fb9dc26fe26caf13bdbf/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/b00e4ad0b80e0c8c757940fafe6219911c1ef9c5/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/db37fc6c23996f7c714a6d6a73dad1a80be29261/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/03b22960a9cfb5c50d881cb8a05efe79a3ff6fc4/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/3966be79061e063f459545f2f6af641e9d94f5e5/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/792a6d47501241ae390b28e0a5ededa44108a30e/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/1a3821fdacea55613f69130f4a7a1c55a1967bb5/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/4db9b827623717880ad02442e23a2e7f4225979e/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/f44188494f867bdfbd39dc797f83cf97577b7c1c/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/af7ba3eb46c867c02c3b9e2b2e847a25b11403a2/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/ade5fc7a94fcb467c5442ca278a29f2523a7814b/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/c23b98bb86d68e99c0b2369c3f89254a006b9370/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/a7675b1842557df0dcd4763191016ef89b889a03/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/6c1e755e1c776ae10dfcdfb11d4980e4f6a4700a/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/5428ca0f824712f1d2ece5ec2981dae9a7a3ddcb/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/c1ec9d190032598b346c09be98e92521030fa643/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/2897cddecca48db95eebb5a1c8a375b8604a585c/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/3546c7b951d82a34337dabb90da0275902c7c01e/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/6661ac1e5e5f587bcdb7b4d2bb0a69fe3c041756/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/324571c3e0d3835ee4f674f1c7c88f08807b32f9/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/9c34055c1e74ce03ccf7554c4825199e/raw/334bc64b73b26b60396bc435eabada37c07beba6/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
@dmitrygx can you pls take a look (force pushed because it's very small fix)

**Pull request merged**